src/Raven.Client/Http/WriteBalanceBehavior.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Raven.Client.Http
+{
+    public enum WriteBalanceBehavior
+    {
+        None,
+        ClientContextSelection
```

Comment:
I do not like this name, but I cannot think of a better one

Reply:
Updated it, let's see if it would work better.